### PR TITLE
add new stack screen

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Create new stack page Flutter Gallery App
## Task
Link of task: https://www.notion.so/dustin0306/Row-Column-Expanded-a6794bc43cc64c2d89f9ec8fac98f8fc?pvs=4
## Screenshot
image
##Videos

https://github.com/krish-do-goldenowl/walter-theory-trainings/assets/144195318/f02d6e93-518a-4674-ac4d-7d7d2b05098f

### CODE REVIEW

File: Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text("Stack & Align")),
+      bottomNavigationBar: viewOption(),
+      body: mainView(),
+    );
+  }
+
+  Widget mainView() {
```
Comment (dustin-goldenowl) [inline Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart]: replace "_mainView()"

File: Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+              Container(color: Colors.green, height: 200, width: 200),
+              Container(color: Colors.orange, height: 120, width: 120),
+            ],
+          ),
+        );
+      },
+    );
+  }
+
+  Widget viewOption() {
```
Comment (dustin-goldenowl) [inline Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart]: replace to "_viewOption()"

Comment (ngoctienUIT) [reply Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart:90]: solved

Comment (WalterGoldenOwl) [reply Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart]: solved

File: Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply Flutter/flutter_gallery_app/lib/src/presentation/stack/screen/stack_page.dart]: solved

**Pull request merged**